### DISCUSSION TITLE
Are GitHub hosted runners going to include the GNU Arm Embedded Toolchain?

### DISCUSSION THREAD

**Discussion by carlosperate** [actions/virtual-environments#6520] [General questions]
Based on this blog post from the 2nd of November that's what I understood:
https://github.blog/2022-11-02-github-partners-with-arm-to-revolutionize-internet-of-things-software-development-with-github-actions/

However, I'm not sure what "GitHub Actions cloud hosted runners" means in this context:

> We’re excited to share that we have partnered with Arm to revolutionize IoT software development by making Arm Development tools (Arm cross-compiler and [Arm Virtual Hardware](http://github.com/Arm-software/AVH-GetStarted)) natively available inside GitHub Actions cloud hosted runners to create an efficient CI workflow.

Is that the GitHub hosted runners? Self hosted runners? Something else entirely?

**Comment by MichaIng** [comment]
As far as I understood, reading through the workflow examples, it uses AWS hosted VMs, which requires an AWS subscription, passing the related steps from the workflow over. It's a bit at the edge of my understanding, so maybe I'm wrong, but so far I don't understand the revolution in this, compared to actual ARM runners, or native ARM emulation + toolchains so that one doesn't need to setup QEMU + container.
